feat: make tenant_id required for source-bing-ads (#65599)
airbytehq/airbyte: docs/integrations/sources/bing-ads-migrations.md

# Bing Ads Migration Guide

## Upgrading to 3.0.0

> This version makes the `tenant_id` field required in the connector configuration. Previously, this field was optional with a default value of "common".

Reviewer: ```suggestion
This version makes the `tenant_id` field required in the connector configuration. Previously, this field was optional but was required.
```
